install/update_0803_083.php

```diff
@@ -1104,14 +1098,14 @@ function update0803to083() {
    // Clean unused slalevels
    $DB->deleteOrDie("glpi_slalevels_tickets", [
          'NOT' => [
-            'glpi_slalevels_tickets.tickets_id' => new \QuerySubQuery([
-               'SELECT' => "glpi_tickets.id",
-               'FROM'   => "glpi_tickets"
-            ])
-         ],
-         'NOT' => [
-            'glpi_slalevels_tickets.slalevels_id' => new \QuerySubQuery([
-               'SELECT' => "glpi_tickets.slalevels_id",
+            new \QueryExpression(
```

Comment:
You cannot use a QueryExpression as array key, you have to put the `IN ...` inside the QueryExpression. If you want, you can build it using a QuerySubQuery `getQuery()` method.